wdl-lint/tests/lints/mixed-indentation/source.wdl

```diff
@@ -0,0 +1,25 @@
+## This is a test of having mixed indentation in commands
+
+version 1.1
+
+task test1 {
+    command <<<
+        this line is prefixed with ~{"spaces and has tailing mixed indentation"}  		
+		this line is prefixed with ~{"tabs"}
+      	this line is prefixed with mixed indentation
+        this line is prefixed with spaces and has trailing mixed indentation  		
+			   >>>
```

Comment:
I think your code should properly handle this case, but I want to see an explicit test for this case
```
        this line is getting ~{
            if foo
            then "split across multiple lines"
            else "by a placeholder"
        } but is all one literal line in the command text
```

Reply:
Follow-up Q: Does escaping a newline do anything to change the parsing? I _think_ it would just be treated as any other literal text, without any special handling. Can we also add a test case for that?
```
        this line won't receive \
 \t         special handling and should be a warning
```

Reply:
I'll add a test case for your first comment.

For your second comment, I think that might be a bug; I'll add a test case for it and fix.

Reply:
I pushed up a change to the test that includes both test cases and fixes the bug in line continuations.